WordPress/Classes/Utility/WPAssetExporter.m

```diff
@@ -40,7 +41,16 @@ - (void)exportAsset:(ALAsset *)asset
    stripGeoLocation:(BOOL)stripGeoLocation
   completionHandler:(void (^)(BOOL success, CGSize resultingSize, NSData *thumbnailData, NSError *error)) handler
 {
-
+    if (!asset.defaultRepresentation) {
+        if (handler) {
+            NSDictionary * userInfo = @{NSLocalizedDescriptionKey:NSLocalizedString(@"The media you are trying to use is not available locally at the moment. If it belongs to a photo stream please try again later.", @"Message that explains to a user that the current asset they selected is not available on the device. This normally happens when user selects a media that belogns to a photostream that needs to be downloaded locally first.")};
```

Comment:
Another way for them to fix it it's to open the image in full screen in the Photos app. That will trigger the download the image locally.

@"This images in Photo Streams at the moment is not available on your device to be added to your WordPress. Try opening it full screen in the Photos app before using it."